### PR TITLE
Completed Gamta Exercise and added notes

### PR DESCRIPTION
**Pull request by samarth25992**
Please review the changes.

### CODE REVIEW

File: Gamta/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles.css"/>
+    <title>Buy healthy snacks online</title>
```
Comment (deadzg) [inline Gamta/index.html:7]: Do you want this to be the title of the page?
The actual site has a different title.

Comment (shahtanv) [inline Gamta/index.html:7]: Do you want this to be the title of the page?
The actual site has a different title.

File: Gamta/styles.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
+body {
+  font-family: Roboto,sans-serif;
```
Comment (shahtanv) [inline Gamta/styles.css:2]: Roboto is not standard font. It wont be applied unless you supply font files to the browser.

File: Gamta/styles.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  margin:0px;
+}
+
+.header {
+  font-size: 16px;
+  font-weight: bolder;
```
Comment (shahtanv) [inline Gamta/styles.css:8]: Is "bolder" a valid value? I think it was "bold"

File: Gamta/styles.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  color: #e1232a;
+  box-shadow: 0px 3px #eee;
+}
+
+.topBar {
+  background: black;
```
Comment (shahtanv) [inline Gamta/styles.css:14]: Its better to use hex values everywhere for consistency.

File: Gamta/styles.css [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  text-align: center;
+  color: white;
+}
+
+.topBar p {
+  margin: 0px;
+  padding: 10px 0px;
```
Comment (shahtanv) [inline Gamta/styles.css:21]: Its better to use hex values everywhere for consistency.

File: Gamta/styles.css [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+.menuBar {
+  width: 80%;
+  margin: auto;
+  padding: 10px 0px;
+  height: 80px;
+}
+
+.menuBar img {
+  width: 15%;
+  height: auto;
```
Comment (shahtanv) [inline Gamta/styles.css:33]: Is this required?

**Comment by shahtanv** [comment]
Generally, Git does not display docx and draw.io files. Are you able to view directly on Git?

Also, hope you have added following in you diagram:

1) Type of protocol
2) CDN and notes about it. What it does, what are famous CDNs available.
3) Proxy server and notes about it. What it does and famous proxy servers available.
4) Load balancer and notes about it. What it does and famous load balancers available.

File: 1st Exercise/styles.css [hunk 7]
```diff
@@ -0,0 +1,7 @@
+li {
+  display: inline;
+  background: #13A4A5;
```
Comment (shahtanv) [inline 1st Exercise/styles.css:3]: Write in lower case.

File: Gamta/index.html [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+  </head>
+  <body>
+    <div class="header">
+      <div class="topBar">
+        <p>Free Delivery on all orders above Rs. 500.</p>
+      </div>
+      <div class="menuBar">
```
Comment (shahtanv) [inline Gamta/index.html:14]: There is one ore bar about the menu bar which displays phone number, email etc...

File: Gamta/index.html (hunk 1 above)
Comment (samarth25992) [reply Gamta/index.html:7]: Yeah, The title should change on the original website also. Let's keep this one.

File: Gamta/styles.css (hunk 2 above)
Comment (samarth25992) [reply Gamta/styles.css:2]: I have added 'Roboto' font using <link> tag.

File: Gamta/styles.css (hunk 3 above)
Comment (samarth25992) [reply Gamta/styles.css:8]: Changed it to bold.

File: Gamta/styles.css (hunk 4 above)
Comment (samarth25992) [reply Gamta/styles.css:14]: Changed the values to hex.

File: Gamta/styles.css (hunk 6 above)
Comment (samarth25992) [reply Gamta/styles.css:33]: Yes, the image is quite big. So I am giving width and height automatically grows.

File: 1st Exercise/styles.css (hunk 7 above)
Comment (samarth25992) [reply 1st Exercise/styles.css:3]: Completed.

File: Gamta/index.html (hunk 8 above)
Comment (samarth25992) [reply Gamta/index.html:14]: I have added the bar.

File: Gamta/styles.css [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+.topBar p {
+  margin: 0px;
+  padding: 10px 0px;
+}
+
+.infoBar {
+  background: #e1232a;
+  list-style-type: none;
```
Comment (shahtanv) [inline Gamta/styles.css:32]: This property should be applied to ul and not div. Its fine, you can change in the next PR.

**Comment by shahtanv** [comment]
Looks good. Go ahead and merge.

**Pull request merged**